src/shared/hooks/useLogin.ts

```diff
@@ -1 +1,75 @@
-//TODO: A hook to get login credentials any time through the app.
+import {
```

Comment:
This file has all the logic to handle a login. 

The main point of it is to return 3 things:
- a login function, that can be called at any time from the app to authenticate a user.
- A logout function, that can be called from anywhere in the app to log out
- A `user` object, that contains all the data for a user. This can be accessed from anywhere in the app.